src/integrations/airtable.ts

```diff
@@ -66,20 +66,24 @@ export const find = async (table: string, filterByFormula?: string, fields?: str
     });
 }
 
-// commenting out because this program should READ-ONLY
+export const update = async (table, data) => {
+    return new Promise((res,rej) => {
+        base(table).update(data, function(err, record) {
```

Comment:
I think it would be much cleaner to replace the promises with async/await in here